data/json/body_parts.json

```diff
@@ -24,7 +24,13 @@
     "base_hp": 60,
     "drench_capacity": 40,
     "smash_message": "You smash the %s with a powerful shoulder-check.",
-    "bionic_slots": 80
+    "safe_bionic_slots": 40,
+    "max_bionic_slots": 80,
+    "faults": [
+      { "effect": "open_skin", "slots_requirement": 40 },
+      { "effect": "nerve_pain", "slots_requirement": 50 },
+      { "effect": "nerve_pain_2", "slots_requirement": 60 }
```

Comment:
```suggestion
      { "effect": "open_skin", "slots_requirement": 60 },
      { "effect": "nerve_pain", "slots_requirement": 40 },
      { "effect": "nerve_pain_2", "slots_requirement": 50 }
```

Reply:
Why?

Reply:
It is a lot less likely that something in the torso is going to be close enough to the surface to cause skin problems (take a look at which CBMs are in the torso). OTOH, there are still a lot of nerves there...